### ISSUE TITLE
[1.12.2] 'Exception ticking world' crash, and a suggestion.

### ISSUE THREAD

**Issue by MechanosG** [ferreusveritas/Dynamic-Trees#282]
**Describe the bug**
Our server suddenly started crashing anytime a certain area was loaded.

**To Reproduce**
Simply enter the area with what I assume would be the bugged out tree in it.  Or in my case since I'm nearby, trying to enter the server immediately causes the crash.

**Additional context**
I use Dynamic Trees and the compatibility mods for Pams Harvestcraft and Thaumcraft, in a modpack currently in private multiplayer testing (along with a lot of other mods).  When the crash started happening, I didn't suspect Dynamic Trees at all and started removing mods 1 at a time, starting with the biggest mods.  After about 6 hours of this, I finally decided to remove Dynamic Trees and the 2 compatibility mods.  The crash stopped happening immediately.  So I restored the server's world from backup (to when it was crashing), re-added all removed mods, then only removed the pams/thaumcraft compatibility mods just to make sure it wasn't one of those.  The crash persisted.  So I removed Dynamic Trees itself, and the crash went away.  So I think it's safe to say Dynamic Trees is *probably* the culprit.

**Crashes**
Here's the mod versions I'm using off of Twitch/Curseforge
DynamicTrees-1.12.2-0.9.5.jar
DynamicTreesPHC-1.12.2-1.4.2.jar
DynamicTreesTC-1.12.2-1.4.1e.jar

Here's one of the many crash logs from the server.
[crash-2019-07-05_16.46.03-server.txt](https://github.com/ferreusveritas/DynamicTrees/files/3364257/crash-2019-07-05_16.46.03-server.txt)

**Mods List**
Crash log should have that info.

-----
I also have a suggestion, unrelated to the above crash.  With regards to the amount of time invested planting trees then harvesting them when DT is installed, I feel the default values are enough to chase away many people from wanting to use DT in their modpacks.  While DT is very aesthetically pleasing, most mods are designed around assuming you can gather wood at a vanilla-like pace.  Some even assume you can get it even faster via tool mods, vein miner-esque mods, or automation.  But the default values with DT, even when mass planting a good 30+ trees spread out over an area, seems to result in having to wait far too long to get wood.  Yes, I am aware that with config files you can increase log output (and that it calls you a cheater lol) and yes you can increase tree growth speed (at your own risk).  But both options feel like they come at a form of consequence.  The latter being fear of increased server lag.

My suggestion is to have the mod default to one DT tree roughly being able to produce the amount of wood a vanilla tree can produce after planting, in the same amount time.  Or even half as much wood, since DT allows you to harvest by cutting only the base.  Even if you have to wait longer on DT trees, but get enough wood from it that mathematically you're still getting the same amount of wood, just with less harvests, the mod would become a lot more user friendly.  I would happily plant twice as many trees in exchange for easy cutting and have to wait twice as long, as long as I'm still getting half-to-equal the wood I'd get from vanilla trees in the same amount of time.  As it stands though, I currently have my modpack set to use a 2x multiplier on wood output and an elevated growth speed (with worldgen trees turned off and auto acorn dropping turned off), and yet I still find myself gathering wood at a snail's pace.  In comparison, a Rubber Tree from GT:CE, which I believe grows slower than a vanilla tree and grows to roughly a vanilla oak in size, drastically out produced my DT trees, no automation and a vanilla stone axe.  I also tried multiple DT tree types in different biomes, finally settling on birch, oak, and dark oak in a beach biome as the soil seemed to not run out of nutrients very quickly with that combination.  And it was just..  really really slow.

That said, I am a fan of DT now.  I hope you understand my concern.  Thanks for the mod.

**Comment by linuskr** [comment]
As i wrote in the probably related issue https://github.com/ferreusveritas/DynamicTrees/issues/214, i think this is a problem with ThutCore.

**Comment by MechanosG** [comment]
Hmm, I see.  Totally forgot I posted this issue.  Unfortunately we had to remove Dynamic Trees back then in order to get the server running again.  Coincidentally, I recently started working on the pack again, and the latest update re-adds Dynamic Trees and several compatibility mods.  I'll post here if we run into the issue again (and I'm able to test it).  If the problem does happen again and you're right about ThutCore, would I be able to theoretically remove that mod and have that game world be loadable again?  Or would the error persist and need some other way of testing?
